### PR TITLE
15407: Storybook Component - EmergencyBanner

### PR DESCRIPTION
**Pull request by craigwheeler**
## Description
Created story for EmergencyBanner component | closes [#15407](https://app.zenhub.com/workspaces/vsp-design-system-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/va.gov-team/15407)

## Acceptance criteria
- [x] All the "stories" found in EmergencyBanner.mdx are written as Storybook stories
- [x] These stories are found in packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx

## Definition of done
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import EmergencyBanner from './EmergencyBanner';
+
+export default {
+  title: 'Library/EmergencyBanner',
+  component: EmergencyBanner,
+  argTypes: {
+    visible: { control: 'boolean' },
+    title: { control: 'text' },
+    content: { control: 'text' },
+  },
```
Comment (cvalarida) [inline packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx:11]: All these are defined in the component's `propTypes`, so I don't think you need these.

File: packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        .
+      </p>
+    </div>`,
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  ...defaultArgs,
+};
```
Comment (cvalarida) [inline packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx:51]:
```suggestion
};

export const ErrorBanner = Template.bind({});
ErrorBanner.args = { ...defaultArgs, type: 'error' };
```

Looks like there's some added functionality if `type === 'error'`. Probably could even add this as a `'select'` in the `argTypes` up top, where the options are `undefined` and `'error'`.

**Review by cvalarida — APPROVED** [review]

File: packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx (hunk 2 above)
Comment (craigwheeler) [reply packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx:51]: good catch! i'll update to add select functionality for type `error` and `undefined` .. thanks!

File: packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx (hunk 1 above)
Comment (craigwheeler) [reply packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx]: updated to show types instead tks

**Pull request merged**